File: SoloLearn Codes/Carrot Cake/CarrotCake.java
import java.util.Scanner;

public class CarrotCake
{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int carrot = sc.nextInt();
        int box = sc.nextInt();
        System.out.println(carrot % box >= 7 ? "Cake Time" : "I need to buy " + (7 - (carrot % box)) + " more");
    }
}

File: SoloLearn Codes/Carrot Cake/carrot_cake.c
#include <stdio.h>

int main() {

    int c, b;
    scanf("%d%d", &c, &b);
    int k = c % b;
    printf((k >= 7) ? "Cake Time" : "I need to buy %d more", 7 - k);
    return 0;
}